src/components/Detail.css

```diff
@@ -0,0 +1,62 @@
+.details-container {
```

Comment:
- [ ] On the mobile view, I couldn't find a way to close the navbar:

![image](https://user-images.githubusercontent.com/57421823/214326533-a2a8f294-96a1-44ce-9677-a4ca650bdb5a.png)


If it is indeed possible to close the navbar, please try to make it more obvious so your users can do it easily.  If this is by design, you may choose to leave it, however, this is **strongly not recommended** because the area for the content is vastly reduced. 👍

Reply:
Well noted! The NavBar bug for the mobile menu is now fixed!
Thanks so much for the Observation and Recommendation!!